src/commonMain/kotlin/io/github/koalaplot/core/xygraph/XYGraph.kt

```diff
@@ -600,51 +679,50 @@ private fun Grid(
 }
 
 private fun DrawScope.drawVerticalGridLines(
-    values: List<Float>,
+    xOffsets: List<Float>,
     scale: Float,
     style: LineStyle?
 ) {
     if (style != null) {
-        values.forEach {
+        xOffsets.forEach {
             drawGridLine(
                 style,
-                start = Offset(it * scale, 0f),
-                end = Offset(it * scale, size.height)
+                Offset(it * scale, 0f),
+                Offset(it * scale, size.height)
             )
         }
     }
 }
 
 private fun DrawScope.drawHorizontalGridLines(
-    values: List<Float>,
+    yOffsets: List<Float>,
     scale: Float,
     style: LineStyle?
 ) {
     if (style != null) {
-        values.forEach {
-            drawGridLine(
-                style,
-                start = Offset(0f, scale - it * scale),
-                end = Offset(size.width, scale - it * scale)
-            )
+        yOffsets.forEach {
+            val yPos = scale - it * scale
+            drawGridLine(style, Offset(0f, yPos), Offset(size.width, yPos))
         }
     }
 }
 
-private fun DrawScope.drawGridLine(gridLineStyle: LineStyle?, start: Offset, end: Offset) {
-    if (gridLineStyle != null) {
-        with(gridLineStyle) {
-            drawLine(
-                start = start,
-                end = end,
-                brush = brush,
-                strokeWidth = strokeWidth.toPx(),
-                pathEffect = pathEffect,
-                alpha = alpha,
-                colorFilter = colorFilter,
-                blendMode = blendMode
-            )
-        }
+private fun DrawScope.drawGridLine(
```

Comment:
If multiple parameters on a single line doesn't violate the line length rules check, please don't split onto multiple lines.